ddp-workspace/projects/ddp-sdk/src/lib/models/activity/activityForm.ts

```diff
@@ -19,9 +19,9 @@ export class ActivityForm {
     public sectionIndex: number | null;
     public statusCode: ActivityStatusCodes;
 
-    public validate(): boolean {
+    public validate(validateOnlyVisibleSections?: boolean): boolean {
         let isValid = true;
-        for (const section of this.getAllSections()) {
+        for (const section of this.getAllSections(validateOnlyVisibleSections)) {
```

Comment:
I'd suggest filtering visible sections here, but not in `getAllSections` method.

Reply:
Thank you for your feedback. Done!